ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt

```diff
@@ -11,59 +11,93 @@ import com.intellij.remoterobot.utils.waitFor
 import org.assertj.swing.timing.Pause
 import java.time.Duration
 
+/**
+ * JTreeFixture designed to work with models that use AsyncTreeModel
+ */
 class JTreeFixture(
     remoteRobot: RemoteRobot,
     remoteComponent: RemoteComponent
 ) : ComponentFixture(remoteRobot, remoteComponent) {
     var separator: String = "/"
 
-    fun hasPath(vararg paths: String) = try {
-        runJsPathMethod("node", *paths)
-        true
-    } catch (e: Exception) {
-        false
+    fun hasPath(vararg path: String): Boolean {
+        val fullPath = path.joinToString(separator)
+        return step("hasPath $fullPath") {
+           callJs(
+                """
+                const jTreeFixture = JTreeFixture(robot, component);
+                jTreeFixture.replaceSeparator('$separator')
```

Comment:
what does replacing the separator accomplish?

Reply:
Support for file browser tree which needs to search for "/", but the default separator is "/" leading to failing. In the file browser fixture we replace the separator to be "|" to solve that.